api/DataTransfer.json

```
@@ -415,7 +415,7 @@
               "version_added": true
             },
             "safari": {
-              "version_added": false
+              "version_added": "11.1"
             },
             "safari_ios": {
               "version_added": false
```

```suggestion
              "version_added": "11.3"
```

mdn-bcd-collector tested false in 10.3 and true in 11.3. (No iOS 11 results available.)

---

A check in BrowserStack shows `false` for iOS 11. 👍

For some reason I had in my head that `DataTransfer` wasn't supported at all on iOS. I'd bet there are lots of wrong `false` values for iOS in this interface.